clustoapi/apps/attribute.py

```diff
@@ -177,6 +178,26 @@ def add_attr(name):
     if not obj:
         return util.dumps(msg, status)
 
+    try:
+        json_kwargs = request.json
+    except ValueError as ve:
+        return util.dumps('%s' % (ve,), 400)
```

Comment:
No, request.json will be blank if there is no json-able body. It _will_ return an error if the json is malformed tho.
Current tests pass with this btw, so I'm confident in that statement.